src/main/java/taxi/controller/car/AddCarController.java

```diff
@@ -30,6 +30,6 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         Manufacturer manufacturer = manufacturerService.get(manufacturerId);
         Car car = new Car(model, manufacturer);
         carService.create(car);
-        resp.sendRedirect(req.getContextPath() + "/cars/add");
+        resp.sendRedirect(req.getContextPath() + "/cars");
```

Comment:
i think we do not need to change our previous end point names
can we add new car in our /car end point?

Reply:
well, the logic behind this is after I create or add something to my DB, I got redirected to the updated list, and there i  have buttons (links) to do this action again (if needed).

Reply:
Again it's not necessary in the task, but I've implemented this. I don't think it's a big deal.

Reply:
i think its ok to improve your solution is you want so, more practice makes perfect 😃